src/megachatapi.h

```diff
@@ -2387,6 +2387,31 @@ class MegaChatApi
      */
     void setChatTitle(MegaChatHandle chatid, const char *title, MegaChatRequestListener *listener = NULL);
 
+    /**
+     * @brief Allows any user to preview an open chat without be part of.
+     *
+     * The associated request type with this request is MegaChatRequest::TYPE_PREVIEW_CHAT_LINK
+     * Valid data in the MegaChatRequest object received on callbacks:
+     * - MegaChatRequest::getChatHandle - Returns the chat identifier
+     * - MegaChatRequest::getText - Returns the title of the chat.
+     *
+     * On the onRequestFinish error, the error code associated to the MegaChatError can be:
+     * - MegaChatError::ERROR_ARGS - If chatlink has not an appropiate format
+     * - MegaChatError::ERROR_EXIST - If the chat link has been opened before in the current session
```

Comment:
`ERROR_EXIST` also in case where the chat is already joined (own user participates). I will change it.